samples/package.json

```diff
@@ -13,9 +14,9 @@
     "test": "c8 mocha --timeout 600000 test/*.js"
   },
   "dependencies": {
-    "@google-analytics/admin": "^0.1.0",
     "google-auth-library": "^6.0.5",
     "google-gax": "^2.6.3",
+    "@google-analytics/admin": "^1.0.0",
```

Comment:
this gets updated automatically on release, it looks like it's okay on the main branch now.

Reply:
Did a rebase with upstream, the version seems fine now.